src/main/java/duke/exception/DoneException.java

```diff
@@ -0,0 +1,13 @@
+package duke.exception;
+
```

Comment:
I think can include a javadocs explaining what causes the exception to be thrown.